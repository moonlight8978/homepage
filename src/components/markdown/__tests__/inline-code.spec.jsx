import React from 'react'
import { render } from '@testing-library/react'

import InlineCode from '../inline-code'

test('renders string', () => {
  const children = '[1, 2].map(number => number * 2)'
  const { container } = render(<InlineCode>{children}</InlineCode>)
  expect(container.firstChild).toMatchSnapshot()
})

test('renders react node', () => {
  const children = <>{'[1, 2].map(number => number * 2)'}</>
  const { container } = render(<InlineCode>{children}</InlineCode>)
  expect(container.firstChild).toMatchSnapshot()
})
