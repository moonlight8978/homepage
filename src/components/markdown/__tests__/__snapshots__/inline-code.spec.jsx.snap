// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders react node 1`] = `
<code
  class="code"
>
  [1, 2].map(number =&gt; number * 2)
</code>
`;

exports[`renders string 1`] = `
<code
  class="code"
>
  [1, 2].map(number =&gt; number * 2)
</code>
`;
